### PR TITLE
added iam policy to the deprecation table

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-7754

Please see my comments in the PR, as I had some questions as I continue to work on this issue.

### CODE REVIEW

File: governance/iam_policy_ctrl.adoc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [#iam-policy-controller]
 = IAM policy controller
+V2.9: Deprecated
```
Comment (jc-berger) [inline governance/iam_policy_ctrl.adoc]: I know that the issue requires me to add the deprecation label to the IAM policy doc. 

I've never added a label to a doc in ascii doc before, so I followed these instructions from AsciiDoctor: https://docs.asciidoctor.org/asciidoc/latest/tables/customize-title-label/ 

Please let me know if I did it incorrectly or if there's an example I can look at.

File: governance/iam_policy_ctrl.adoc [hunk 2]
```diff
@@ -39,6 +41,10 @@ View the following parameter table for descriptions:
 |===
 | Field | Optional or required | Description
 
+| `IAM policy controller`
+| Required
+| Runs on your managed cluster, monitors the roles of the users of the cluster, and notifies you of non-compliant policies.
```
Comment (jc-berger) [inline governance/iam_policy_ctrl.adoc]: SMEs, I didn't know if the policy controller were required and what description would be best.

Please review.

File: governance/iam_policy_ctrl.adoc (hunk 1 above)
Comment (dockerymick) [inline governance/iam_policy_ctrl.adoc]:
```suggestion
= IAM policy controller (Deprecated)
```

File: governance/iam_policy_ctrl.adoc [hunk 3]
```diff
@@ -1,5 +1,7 @@
 [#iam-policy-controller]
 = IAM policy controller
+V2.9: Deprecated 
+:version-label: Deprecated
```
Comment (dockerymick) [inline governance/iam_policy_ctrl.adoc]:
```suggestion
```

File: governance/iam_policy_ctrl.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/iam_policy_ctrl.adoc]: This row isn't required. It seems repetitive to what is explained in the overview, but feel free to improve the explanation is needed.

**Review by dockerymick — CHANGES_REQUESTED** [review]
I checked the preview of the file to see if the ASCII Doc label would render. Nothing appeared, so I suggest removing it. We typically do not use ASCII labels
Screenshot 2023-10-02 at 2 54 37 PM

File: governance/iam_policy_ctrl.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/iam_policy_ctrl.adoc]: Here is the link to the deprecations table: https://github.com/stolostron/rhacm-docs/blob/2.9_stage/release_notes/deprecate_remove.adoc#product-title-short-deprecations

Comment (dockerymick) [reply governance/iam_policy_ctrl.adoc]: Here is also an example of adding the entry https://github.com/stolostron/rhacm-docs/pull/5552/files#diff-bddfa6e67ebe79d7e667fa5c1f09c323997d02702dbb350aab05029f5802575aR24-R29

Comment (jc-berger) [reply governance/iam_policy_ctrl.adoc]: @dockerymick thanks for the feedback! Are you saying the IAM policy controller is not required. Therefore, should it even be in the table if it's not required and the explanation is redundant? 

Or do you mean, the explanation is redundant, and that's not required?

File: governance/iam_policy_ctrl.adoc [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [#iam-policy-controller]
-= IAM policy controller
+= IAM policy controller
```
Comment (dockerymick) [inline governance/iam_policy_ctrl.adoc]:
```suggestion
= IAM policy controller (Deprecated)
```

File: governance/iam_policy_ctrl.adoc [hunk 5]
```diff
@@ -39,6 +39,9 @@ View the following parameter table for descriptions:
 |===
 | Field | Optional or required | Description
 
+| `IAM policy controller`
+|  Optional 
+
```
Comment (dockerymick) [inline governance/iam_policy_ctrl.adoc]: The entire entry here should be removed. Add a new row to the [Deprecations table](https://github.com/stolostron/rhacm-docs/blob/2.9_stage/release_notes/deprecate_remove.adoc#product-title-short-deprecations) for IAM policy controller
```suggestion
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: governance/iam_policy_ctrl.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/iam_policy_ctrl.adoc]: Yes, it's redundant. It is best to revert the changes bc this table doesn't need to be updated

**Review by ch-stark — APPROVED** [review]
lgtm thank you

File: release_notes/deprecate_remove.adoc [hunk 6]
```diff
@@ -156,6 +156,12 @@ A _deprecated_ component, feature, or service is supported, but no longer recomm
 |===
 | Product or category | Affected item | Version | Recommended action | More details and links
 
+| Governance
+| IAM policy controller
+| 2.9
+| None
+|
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc:163]: @jc-berger this is the table that I was referring to in my [comment](https://github.com/stolostron/rhacm-docs/pull/5572#discussion_r1344353718)

**Review by dockerymick — APPROVED** [review]
Left a comment for clarification and made an update. the table looks good when I select to view the Deprecation file. Let me know if you still have questions.

**Review by dockerymick — APPROVED** [review]
Left a hidden comment to be addressed. Otherwise, this looks good

File: governance/policy_controllers.adoc [hunk 7]
```diff
@@ -4,7 +4,7 @@
 Policy controllers monitor and report whether your cluster is compliant with a policy. Use the {product-title} policy framework by using the out-of-the-box policy templates to apply policies managed by these controllers. The policy controllers manage Kubernetes custom resource definition (CRD) instances.
 
 Policy controllers monitor for policy violations, and can make the cluster status compliant if the controller supports the enforcement feature. View the following topics to learn more about the following {product-title} policy controllers:
-
+//we should also add the deprecation label here 
 * xref:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller]
 * xref:../governance/cert_policy_ctrl.adoc#certificate-policy-controller[Certificate policy controller]
 * xref:../governance/iam_policy_ctrl.adoc#iam-policy-controller[IAM policy controller]
```
Comment (dockerymick) [inline governance/policy_controllers.adoc]:
```suggestion

* xref:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller]
* xref:../governance/cert_policy_ctrl.adoc#certificate-policy-controller[Certificate policy controller]
* xref:../governance/iam_policy_ctrl.adoc#iam-policy-controller[IAM policy controller (Deprecated)]
```

**Comment by dockerymick** [comment]
I suggest to update the title of the PR to "added iam policy to the deprecations table" @jc-berger

**Pull request merged**